src/OpenSage.Game/Logic/Object/Behaviors/SlowDeathBehavior.cs

```diff
@@ -29,7 +29,7 @@ internal SlowDeathBehavior(SlowDeathBehaviorModuleData moduleData)
             _moduleData = moduleData;
         }
 
-        internal bool IsApplicable(DeathType deathType) => _moduleData.DeathTypes?.Get(deathType) ?? false;
+        internal bool IsApplicable(DeathType deathType) => _moduleData.DeathTypes?.Get(deathType) ?? true;
```

Comment:
Why you changed this to true? I think the default must be **false**

Reply:
Because some models, like e.g. the hero statue of Rohan in bfme only have a single slowDeathBehavior which does not define any death types.